### PR TITLE
S1SingleScatter Cut Version 0 Added

### PR DESCRIPTION
**Pull request by jpienaar13**
Added S1SingleScatter Cut

### CODE REVIEW

File: lax/lichens/sciencerun0.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+00# -*- coding: utf-8 -*-
```
Comment (pdeperio) [inline lax/lichens/sciencerun0.py]: remove "00"?

**Review by pdeperio — CHANGES_REQUESTED** [review]

File: lax/lichens/sciencerun0.py [hunk 2]
```diff
@@ -249,3 +251,42 @@ def _process(self, df):
...
+    It should be applicable to data regardless of if it is ER or NR.
+
+    Author: Jacques (jpienaa@purdue.edu).
+    """
+
+    version=0
+
+    def process(self, df):
+
+        s2width=S2Width()
```
Comment (tunnell) [inline lax/lichens/sciencerun0.py]: Why do you have to instantiate this if it's a static method above?  Also, try not to overload process but instead _process.    atl_rel_width should get loaded into Temp.

**Review by tunnell — CHANGES_REQUESTED** [review]

**Review by tunnell — APPROVED** [review]

**Comment by tunnell** [comment]
This ready?

**Comment by jpienaar13** [comment]
Tested it. It appears to work. So yes?

**Comment by tunnell** [comment]
@pdeperio approve?

**Review by pdeperio — APPROVED** [review]

**Pull request merged**